documentation/asciidoc/computers/raspberry-pi/boot-eeprom-rpi4.adoc

```diff
@@ -1,6 +1,6 @@
 == Raspberry Pi 4 Boot EEPROM
 
-The Raspberry Pi 4 has an SPI-attached EEPROM (4MBits/512KB), which contains code to boot up the system and replaces `bootcode.bin` previously found in the boot partition of the SD card. Note that if a `bootcode.bin` is present in the boot partition of the SD card in a Pi 4, it is ignored.
+Raspberry Pi 4 and 400 computers use an EEPROM to boot the system. All other models of Raspberry Pi computer use the `bootcode.bin` file located in the boot filesystem.
```

Comment:
Didn't we once decide that "Raspberry Pi 4" was a suitable substitute for "Raspberry Pi 4 Model B, Raspberry Pi 400 and Compute Module 4" ? :shrug:  (but of course ultimately this is an @aallan decision)

Reply:
> Didn't we once decide that "Raspberry Pi 4" was a suitable substitute for "Raspberry Pi 4 Model B, Raspberry Pi 400 and Compute Module 4" ? 🤷 (but of course ultimately this is an @aallan decision)

Yes, but in this case I'm trying not to get into a situation where someone with a Pi 400 thinks the Pi 4 stuff doesn't apply to their Pi 400: the next sentence says all `other Pi models...`. I could enumerate all the other Pi models, but I think it's better to specifically call out the two models that are the exceptions rather than list all of the other models which are not.

Reply:
CM4 also uses an EEPROM.

Reply:
> CM4 also uses an EEPROM.

I was deliberately not mentioning CM4 since traditionally the documentation does not specifically refer to compute modules in pages which are not specific to the compute module family of devices: upon reflection this is probably a Bad Thing &trade;

Reply:
Maybe, but the Pi documentation is [over here](https://www.raspberrypi.org/documentation/computers/raspberry-pi.html), while the CM documentation is still [over there](https://www.raspberrypi.org/documentation/computers/compute-module.html), and will probably continue to be separated. Mainly due to the different audiences for the two products.